examples/customresourceinterpreter/README.md

```diff
@@ -86,9 +113,63 @@ spec:
 kubectl --kubeconfig $HOME/.kube/karmada.config --context karmada-apiserver apply -f workload-crd-cpp.yaml
 ```
 
-#### Step2: Deploy webhook configuration in karmada-apiserver
+#### Step2: Deploy webhook configuration in `karmada-apiserver`
+
+We can tell Karmada how to access the resource interpreter webhook service by configuring `ResourceInterpreterWebhookConfiguration`. The configuration template is as follows:
+
+```yaml
+apiVersion: config.karmada.io/v1alpha1
+kind: ResourceInterpreterWebhookConfiguration
+metadata:
+  name: examples
+webhooks:
+  - name: workloads.example.com
+    rules:
+      - operations: [ "InterpretReplica","ReviseReplica","Retain","AggregateStatus", "InterpretHealth", "InterpretStatus", "InterpretDependency" ]
+        apiGroups: [ "workload.example.io" ]
+        apiVersions: [ "v1alpha1" ]
+        kinds: [ "Workload" ]
+    clientConfig:
+      url: https://{{karmada-interpreter-webhook-example-svc-address}}:443/interpreter-workload
+      caBundle: {{caBundle}}
+    interpreterContextVersions: [ "v1alpha1" ]
+    timeoutSeconds: 3
+```
+
+If you only need to access the resource interpreter webhook service in the `Karmada-host` cluster, you can directly configure `clientConfig` with the Service domain name in the cluster:
+
+```yaml
+clientConfig:
+  url: https://karmada-interpreter-webhook-example.karmada-system.svce:443/interpreter-workload
```

Comment:
I'm not sure but is `.svce:443` right?

Reply:
Nice finding, a typo here, it should be `.svc:443`